region/maridia/inner-pink.json

```diff
@@ -2736,6 +2816,7 @@
                   "name": "Shinespark",
                   "notable": false,
                   "requires": [
+                    "canSuitlessMaridia",
```

Comment:
I don't see this as a need to know tech to do the strat.
But there could be a divide on number of frames remaining because it does take longer to turn around without gravity.  But this looks like a vertical spark and not diagonal 🤔

Reply:
canSuitlessMaridia is a very weird tech. It probably should be added and removed from many locations. I dont even think some of the harder underwater tech in the Moat use it ... I'll remove it from here for now, to not introduce an unrelated and unsure change to this PR.